### PR TITLE
Updates reportback view structure

### LINKED ISSUE #4728: bring back previous view of /node/[nid]/rb/reviewed
image

honestly can't recall off the top of my head everything that was included but resembled the inbox view

not sure when this broke but must have been since the last prod deploy

image

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #4728 

Had to re-add some of the fields back in & re-arrange them, based the config off the UK version 

@angaither

### CODE REVIEW

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module [hunk 1]
```diff
@@ -740,7 +740,7 @@ function dosomething_reportback_views_pre_render(&$view) {
     $nid = $view->args[0];
     $quantity = (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quantity') + (int) dosomething_helpers_get_variable('node', $nid, 'sum_rb_quantity_override');
     // Add total quantity to view.
-    $view->attachment_before = '<h2> Total reportback quantity: ' . $quantity . '</h2>';
+    $view->attachment_before = '<h2> Total reportback quantity TEST: ' . $quantity . '</h2>';
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module]: TEST code!

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc [hunk 2]
```diff
@@ -344,7 +344,7 @@ function dosomething_reportback_views_default_views() {
     t('RBID'),
     t('.'),
     t('Campaign'),
-    t('Image'),
+    t('Rendered Reportback Item'),
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc]: let's keep this title image still!

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc [hunk 3]
```diff
@@ -115,17 +117,15 @@ function dosomething_reportback_views_default_views() {
   $handler->display->display_options['fields']['nid']['link_to_entity'] = 1;
   $handler->display->display_options['fields']['nid']['view_mode'] = 'full';
   $handler->display->display_options['fields']['nid']['bypass_access'] = 0;
-  /* Field: Reportback file: Rendered Reportback file */
+  /* Field: Reportback Item: Rendered Reportback Item */
   $handler->display->display_options['fields']['rendered_entity']['id'] = 'rendered_entity';
-  $handler->display->display_options['fields']['rendered_entity']['table'] = 'views_entity_reportback_file';
+  $handler->display->display_options['fields']['rendered_entity']['table'] = 'views_entity_reportback_item';
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc:122]: Sorry :cry:

File: lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc [hunk 4]
```diff
@@ -93,12 +93,14 @@ function dosomething_reportback_views_default_views() {
   $handler->display->display_options['relationships']['nid']['field'] = 'nid';
   $handler->display->display_options['relationships']['nid']['relationship'] = 'rbid';
   $handler->display->display_options['relationships']['nid']['required'] = TRUE;
-  /* Field: Reportback file: Reportback file fid */
+  /* Field: Reportback Item: Reportback file fid */
   $handler->display->display_options['fields']['fid']['id'] = 'fid';
-  $handler->display->display_options['fields']['fid']['table'] = 'views_entity_reportback_file';
+  $handler->display->display_options['fields']['fid']['table'] = 'views_entity_reportback_item';
```
Comment (weerd) [inline lib/modules/dosomething/dosomething_reportback/dosomething_reportback.views_default.inc:98]: Sorry :cry:

**Comment by itsjoekent** [comment]
diego its alright haha

**Comment by itsjoekent** [comment]
@angaither alright I think I covered everything

**Comment by angaither** [comment]
:+1:

**Pull request merged**